staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -1283,6 +1283,19 @@ func TestDefaultDescribers(t *testing.T) {
 	if !strings.Contains(out, "foo") {
 		t.Errorf("unexpected output: %s", out)
 	}
+
+	var ssReplicas int32
+	ss := &appsv1.StatefulSet{
+		ObjectMeta: metav1.ObjectMeta{Name: "foo"},
+		Spec:       appsv1.StatefulSetSpec{Replicas: &ssReplicas},
+	}
+	out, err = DefaultObjectDescriber.DescribeObject(ss)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	if !strings.Contains(out, "foo") {
+		t.Errorf("unexpected output: %s", out)
```

Comment:
Can you change this string so that it matches what the test does? maybe something like:
```suggestion
		t.Errorf("missing `foo` in output: %s", out)
```

Reply:
Done! Addressed the remaining tests in `TestDefaultDescribers` while I was already editing this one.

Reply:
thanks, I didn't realize this was done throughout the file, that's still an improvement ;-)